packages/lib/modules/tokens/approvals/permit2/usePermit2ApprovalSteps.tsx

```diff
@@ -120,9 +123,14 @@ export function usePermit2ApprovalSteps({
 
     // Check if the token has been approved
     const isComplete = () => {
-      const isNotExpired = !!expirations && expirations[tokenAddress] > getNowTimestampInSecs()
-      const isAllowed = allowanceFor(tokenAddress) >= amountToApprove
-      return requiredRawAmount > 0n && isAllowed && isNotExpired
+      const isNotExpired = !!expirations && expirations[tokenAddress] > nowInSecs
+      const isAllowed = allowanceFor(tokenAddress) >= requiredRawAmount
+
+      // some expirations were set too far in the future so redo them
+      // the expiration shouldn't be more than 3 days from now
+      const hasValidExpiry = !!expirations && expirations[tokenAddress] < permitExpiry
```

Comment:
not sure I understand all of this, but why were some expirations set more than 3 days into the future? 
don't we set the expiration dates?